frontend/assets/detail_headers/1_auto.svg

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Where are these SVGs from? Is the license ok?

Reply:
Selfmade, so yes.